src/components/TodoItems.jsx

```diff
@@ -0,0 +1,81 @@
+import { categoryList, priorityList } from '../constants'
+import './todo-items.css'
+
+export default function TodoItems(props) {
+    const styles = {
+        textDecoration: 'line-through'
+    }
```

Comment:
Let's move style to the CSS file and use the className in conditional.

```css
text-linethrough: {
  textDecoration: 'line-through'
} 
```

in JSX:
```jsx
className={item?.isDone ? 'text-linethrough' : ''}
```